src/sentry/monitors/models.py

```diff
@@ -391,6 +391,7 @@ class MonitorCheckIn(Model):
     # The time that we mark an in_progress check-in as timeout. date_added + max_runtime
     timeout_at = models.DateTimeField(null=True)
     monitor_config = JSONField(null=True)
+    trace_id = models.CharField(max_length=32, null=True)
```

Comment:
https://develop.sentry.dev/sdk/event-payloads/contexts/#trace-context

this is the value we are storing

Reply:
should this just be `UUIDField(null=True)` instead?

Reply:
Yeah, a UUIDField probably makes more sense, and is more space efficient, which kind of does matter on this table since it's pretty large.